packages/ui-chart-components/src/components/Axes/XAxis.tsx

```diff
@@ -41,13 +41,13 @@ const renderXAxisLabel = (
   label: string | undefined,
   fillColor: string | undefined,
   isEnlarged: boolean,
-  isExporting: boolean,
+  isExporting: boolean
 ): LabelProps | undefined => {
-  if (label && isEnlarged && !isExporting) {
+  if (label && isEnlarged) {
     return {
       value: label,
       fill: fillColor,
-      offset: -5,
+      offset: isExporting ? -50 : -5,
```

Comment:
This is intended to prevent overlapping between tick labels and the x-axis. Given the tick labels are restricted to a maximum number of characters (using ellipses for longer text) this approach should resolve the issue.